common/api/summary/core-quantity.exports.csv

```diff
@@ -20,6 +20,7 @@ beta;enum;FormatTraits
 beta;function;formatTraitsToArray
 beta;enum;FormatType
 beta;function;formatTypeToString
+deprecated;function;formatTypeToString
```

Comment:
are there any deprecation lint warnings you found when you ran `rush lint`? I assume ecschema-metadata package would have some references